src/flick/friend/tests/test_friendship.py

```diff
@@ -0,0 +1,80 @@
+    USERNAMES = ["alanna", "vivi", "olivia"]
+    SOCIAL_ID_TOKENS = ["test1", "test2", "test3"]
+
+    def setUp(self):
+        for i, name in enumerate(self.USERNAMES):
+            self._create_user(name, self.SOCIAL_ID_TOKENS[i])
+
+    def _create_user(self, username, social_id_token):
+        request_data = {
+            "username": username,
+            "first_name": "test_user",
+            "last_name": "test_user",
+            "profile_pic": self.BASE64_IMAGE,
+            "social_id_token": social_id_token,
+            "social_id_token_type": "test",
+        }
+        response = self.client.post(self.REGISTER_URL, request_data)
+        self.assertEqual(response.status_code, 200)
+
+    def _login_user(self, username, social_id_token):
+        request_data = {"username": username, "social_id_token": social_id_token}
+        response = self.client.post(self.LOGIN_URL, request_data)
+        self.assertEqual(response.status_code, 200)
+        token = json.loads(response.content)["data"]["auth_token"]
+        self.assertIsNotNone(token)
+        return token
+
+    def test_list_friends(self):
+        token = self._login_user("alanna", "test1")
+        auth_headers = {"HTTP_AUTHORIZATION": "token " + token}
+        response = self.client.get(self.FRIEND_LIST_URL, **auth_headers)
+        self.assertEqual(response.status_code, 200)
+
+    def test_send_friend_request(self):
+        token = self._login_user("alanna", "test1")
+        auth_headers = {"HTTP_AUTHORIZATION": "token " + token}
+        request_data = {
+            "user_ids": [2, 3],
+        }
+        response = self.client.post(
+            self.FRIEND_REQUEST_URL, request_data, content_type="application/json", **auth_headers
```

Comment:
good job!! However, I think this would look cleaner if you took advantage of rest framework tools. 
You can instead create a setup method (that runs before each test):
```
def setUp(self):
        self.client = APIClient()
```
and because this is a special api client that is provided by rest_framework (`from rest_framework.test import APIClient`), we can use it like so:
```
        self.client.credentials(HTTP_AUTHORIZATION="Token " + token)
        rating_data = {"user_rating": rating}
        response = self.client.post(self.SHOW_DETAIL_URL, rating_data, format="json")
        content = json.loads(response.content)
```
if you notice, setting `self.client.credentials` will mean that all requests after that request will be including the auth header -- so you don't have to create and pass in an auth header, you can just set `self.client.credentials`.